src/Sylius/Behat/Context/Transform/ProductContext.php

```diff
@@ -41,14 +41,15 @@ public function __construct(ProductRepositoryInterface $productRepository)
      */
     public function getProductByName($productName)
     {
-        $product = $this->productRepository->findOneByName($productName);
+        $products = $this->productRepository->findByName($productName, 'en_US');
```

Comment:
In fact, we should use `%locale%` parameter, but I've got no idea how to pass it here right now.